src/util/logging.c

```diff
@@ -16,8 +17,11 @@ void LOGGING_INIT(int level) {
 }
 
 void LogCallback(const char *level, const char *fmt, ...) {
+  char msg[LOG_MAX_LEN];
+
   va_list ap;
   va_start(ap, fmt);
-  RedisModule_Log(RSDummyContext, level, fmt, ap);
+  vsnprintf(msg, sizeof(msg), fmt, ap);
```

Comment:
Out of curiosity, is the `sizeof(msg)` becomes a no-op in the binaries? We can pass the `LOG_MAX_LEN` as easily

Reply:
yes